go/pkg/cas/upload.go

```diff
@@ -544,6 +545,11 @@ func (u *uploader) visitPath(ctx context.Context, absPath string, info os.FileIn
 		}
 	}
 
+	// Ignore domain sockets (as used by git fsmonitor(.
+	if info.Mode()&os.ModeSocket == os.ModeSocket {
+		return nil, nil
```

Comment:
If you’re ok with skipping all files with unexpected modes, we can do that.

Reply:
I believe that should be backwards compatible. I'm trying to think of a use-case where it would be useful to fail, but nothing comes to mind. I think it is a reasonable change.

Reply:
Done.